site/content/docs/5.2/forms/validation.md

```diff
@@ -169,6 +185,13 @@ For invalid fields, ensure that the invalid feedback/error message is associated
 
 To fix [issues with border radius](https://github.com/twbs/bootstrap/issues/25110), input groups require an additional `.has-validation` class.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This form variant can not be used because it does not respect the Orange Design System specifications.
```

Comment:
The problems mentioned by the design were:

- this form is a module and not a unitary component and that unitary components shall be used to create this same form
- the "Agree to terms and conditions" checkbox shall always be on top of the form". This point had been pointed to be added in the DSM Pages/Form examples. It could be good to make this point confirmed again by designers.